docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_TextInput.htm

```diff
@@ -11,8 +11,9 @@
 <h1>TextInput</h1>
 
 
-TextInput component is decorator for input elements.
+<p>TextInput component is decorator for input elements.</p>
 
+<img src="../../images/mobile_textInput.png" width="15%">
```

Comment:
The image is broken. There should be colored list in background.

Reply:
Fixed